matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/RoomService.kt

```diff
@@ -40,6 +40,18 @@ interface RoomService {
      */
     suspend fun createRoom(createRoomParams: CreateRoomParams): String
 
+    /**
+     * Create a room locally.
+     * This room will not be synchronized with the server and will not come back from the sync, so all the events related to this room will be generated
+     * locally.
+     */
+    suspend fun createLocalRoom(createRoomParams: CreateRoomParams): String
```

Comment:
thinking out loud about about the api design~ not a blocker!

from a consumer api perspective, are local rooms semantically the same as _normal_ rooms? / would we handle interacting with rooms differently if they're normal vs local?

I'm wondering if `CreateRoomParams` could have a `isLocal` option and reusing `createRoom` and `leaveRoom` :thinking: